### ISSUE TITLE
Do we know why the nativeQuery projections fail?

### ISSUE THREAD

**Issue by upiw** [roberthunt/spring-data-native-query-projection#2]
I have just run into this problem in my project.
I found your example whilst googling the problem.

I see your comment that the nativeQuery tests fail!  

Any idea why or how to get around this without resorting to returning Collection<Object[]> ?

**Comment by roberthunt** [comment]
Yeah, this was a demonstration project for a Spring Data bug that I raised: https://jira.spring.io/browse/DATAJPA-980

It has now been fixed but it relies on some changes in Hibernate so you need to be running version Hibernate 5.2.11 or newer with Spring Data 1.10.12 (Hopper SR12), 1.11.8 (Ingalls SR8) or 2.0 GA (Kay)

**Comment by xtremebiker** [comment]
Still can't make it work with native query, would you mind updating the project in order to provide newer dependency versions and fix the tests? I might provide a pull request myself as well.
